common/src/main/java/com/microsoft/identity/common/internal/controllers/LocalMSALController.java

```diff
@@ -316,25 +317,49 @@ public AcquireTokenResult acquireTokenSilent(
         // subsequent CacheRecords represent other profiles (projections) of this principal in
         // other tenants. Those tokens will be 'sparse', meaning that their AT/RT will not be loaded
         final ICacheRecord fullCacheRecord = cacheRecords.get(0);
-
-        if (accessTokenIsNull(fullCacheRecord)
+        if (fullCacheRecord.getAccessToken().refreshOnIsActive()) {
+            Logger.info(
+                    TAG,
+                    "RefreshIn is active. This will extend your token usage in the rare case servers are not available."
+            );
+        }
+        if (parameters.isRefreshIn()) { //refresh_in second call
```

Comment:
What do we mean by second call?

Reply:
Replaced with:    //Note that parameters.isRefreshDueToRefreshIn() == true is always the result of a subsequent silentCall, meaning after the regular/initial silentCall that is normally triggered by user